modules/apps/static/portal-file-install/portal-file-install-test/src/testIntegration/java/com/liferay/portal/file/install/deploy/test/FileInstallConfigTest.java

```diff
@@ -155,7 +155,7 @@ public void testConfigurationDeprecatedFileExtension() throws Exception {
 			".testConfigurationDeprecatedFileExtension");
 
 		String content = "testKey=\"testValue\"";
-		String contentDeprecated = "testKeyDeprecated=\"testValueDeprecated\"";
+		String contentDeprecated = "testKeyDeprecated=testValueDeprecated";
```

Comment:
@tinatian this should be correct, since the one with `String content = "testKey=\"testValue\""` is used as a dummy to make sure the deprecated config has time to load, if it were to fail and accidently load by waiting for a real configuration created after to finish loading. We used a countDownLatch with a .config file which uses that format.